src/Processors/Executors/ExecutionThreadContext.cpp

```diff
@@ -70,6 +71,7 @@ static void executeJob(ExecutingGraph::Node * node, ReadProgressCallback * read_
 
 bool ExecutionThreadContext::executeTask()
 {
+    OpenTelemetrySpanHolder span("ExecutionThreadContext::executeTask() " + node->processor->getName());
```

Comment:
Seems like it creates too many spans, so `00974_query_profiler` sometimes fail with 
```
 <Error> SystemLog (system.opentelemetry_span_log): Queue is full for system log 'DB::OpenTelemetrySpanLog' at 955525
```
Example: https://s3.amazonaws.com/clickhouse-test-reports/0/e13dc23d344454ecda401df4b23b696a691421fe/stateless_tests__release__databaseordinary__actions_.html
More failures: https://play.clickhouse.com/play?user=play#c2VsZWN0IAp0b1N0YXJ0T2ZEYXkoY2hlY2tfc3RhcnRfdGltZSkgYXMgZCwKY291bnQoKSwgZ3JvdXBVbmlxQXJyYXkocHVsbF9yZXF1ZXN0X251bWJlciksICBhbnkocmVwb3J0X3VybCkKZnJvbSBjaGVja3Mgd2hlcmUgJzIwMjItMDQtMDEnIDw9IGNoZWNrX3N0YXJ0X3RpbWUgYW5kIHRlc3RfbmFtZSBsaWtlICclMDA5NzRfcXVlcnlfcHJvZmlsZXIlJyBhbmQgdGVzdF9zdGF0dXMgaW4gKCdGQUlMJywgJ0ZMQUtZJykgZ3JvdXAgYnkgZCBvcmRlciBieSBkIGRlc2M=

@qoega, should we revert it?

Reply:
>More failures: https://play.clickhouse.com/play?user=play#c2VsZWN0IAp0b1N0YXJ0T2ZEYXkoY2hlY2tfc3RhcnRfdGltZSkgYXMgZCwKY291bnQoKSwgZ3JvdXBVbmlxQXJyYXkocHVsbF9yZXF1ZXN0X251bWJlciksICBhbnkocmVwb3J0X3VybCkKZnJvbSBjaGVja3Mgd2hlcmUgJzIwMjItMDQtMDEnIDw9IGNoZWNrX3N0YXJ0X3RpbWUgYW5kIHRlc3RfbmFtZSBsaWtlICclMDA5NzRfcXVlcnlfcHJvZmlsZXIlJyBhbmQgdGVzdF9zdGF0dXMgaW4gKCdGQUlMJywgJ0ZMQUtZJykgZ3JvdXAgYnkgZCBvcmRlciBieSBkIGRlc2M=

not all of them. Probably some.

Too many spans is not the reason we have sanitiser alerts. For 00974_query_profiler we better just disable opentelemetry_span_log with 'set opentelemetry_start_trace_probability=1'. It is enabled there by random settings fuzzer.

I will build asan debug build and will try to reproduce sanitiser alert.

Reply:
> Too many spans is not the reason we have sanitiser alerts. 

Yes, most likely it's two independent issues. 

> For 00974_query_profiler we better just disable opentelemetry_span_log with 'set opentelemetry_start_trace_probability=1'.

Are you sure that it will not produce to many spans in a production environment?